corobo/docker-compose.yml

```diff
@@ -4,3 +4,10 @@ services:
     image: meetmangukiya/corobo
     restart: always
     env_file: .secrets
+    links:
+      - answers
+  answers:
+    image: meetmangukiya/corobo-answers
+    restart: always
+    environment:
+      - ANSWER_END=http://0.0.0.0:8000
```

Comment:
it is set by default, see https://github.com/coala/corobo/blob/master/answers/Dockerfile#L17